### DISCUSSION TITLE
Simple Swarm treasury tokenomics Excel

### DISCUSSION THREAD

**Discussion by stephen-rowan** [Catalyst-Swarm/Catalyst-Beehive#230] [General]
## Simple Swarm treasury tokenomics Excel 
- [x] Simple Swarm treasury tokenomics Excel @whitevo  (Discord https://discordapp.com/channels/856453826060091422/859114652364177429/863334816090554378 )
- [ ] The excel itself is ready we just need to sit down and talk about it.

**Comment by whitevo** [comment]
The excel itself is ready we just need to sit down and talk about it.

**Comment by stephen-rowan** [comment]
Cool - just cleaning up the Project Board Issues. 
Moved to "In Progress"
Added your comment as a "To do" box above (you should be able to do this as well).

**Comment by whitevo** [comment]
yep ok, added the Agenda label back. So it makes it easier to find these tasks where we might need a group discussion

**Comment by stephen-rowan** [comment]
- I took the Agenda label off because it was being discussed that only @Sapereaude7 (Jacob D) could approve and manage Agenda Items, 
- I am working out a taxonomy here https://github.com/Catalyst-Swarm/Catalyst-Beehive/projects/3#card-64495520
- And starting to document the way we want to work here (https://catalyst-swarm.gitbook.io/catalyst-swarm-genesis/governance/a-github-tracking-system#catalyst-swarm-weekly-tasks-project-board ) 
- But all this is a work in progress for Core Swarm of course, :)

**Comment by whitevo** [comment]
Ah ok, then we need to discuss how can I suggest Agenda items because else can't do anything with this task and just sits on my assignee board.
Should I assign to @Sapereaude7

**Comment by stephen-rowan** [comment]
An assignee workflow could be one approach. I posted a comment in Swarm Discord to draw attention to how we should manage Agenda Items (as opposed to Action Items and general Issues).
https://discord.com/channels/856453826060091422/856453826932768770/867338704506978355

**Comment by stephen-rowan** [comment]
Raised this Issue - Action Item : How to manage Agenda Items #58 and allocated To Jacob D

**Comment by Seomon4u** [comment]
This issue was discussed in -> https://github.com/Catalyst-Swarm/Catalyst-Beehive/issues/78
